server/svix-server/src/v1/endpoints/application.rs

```diff
@@ -349,8 +349,8 @@ async fn patch_application(
     Ok(Json((app, metadata).into()))
 }
 
-const DELETE_APPLICATION_DESCRIPTION: &str = "Delete an application.";
-
+/// Delete an application.
+#[aide_annotate(op_id = "delete_application_api_v1_app__app_id___delete")]
```

Comment:
So it takes the description it from the comment?

Is there a way to automatically generate the op_id? E.g. if you do it on the router itself?

I'd much rather have something that's automatically generated than something we need to manually update like this. Even at the expense of compatibility with what we have in the op_id.

Reply:
There is a default, it picks the name of the function (and just that, nothing else) if you don't specify one. But I also put in an override as seen here, so we can use the existing operation IDs for operations that already exist. There are a few operations in rust which don't exist in the OpenAPI (some `PATCH` ones), you can see those don't set an `op_id` by hand.

Reply:
OK, we can always worry about this later if we have other thoughts/concerns.